src/biokbase/narrative/jobs/job.py

```diff
@@ -341,16 +330,18 @@ def _update_state(self, state: dict) -> None:
         # Check if there would be no change in updating
         # i.e., if state <= self._acc_state
         if self._acc_state is not None:
-            if {**self._acc_state, **state} == self._acc_state:
+            if merge(self._acc_state, state) == self._acc_state:
```

Comment:
It might be an idea to remove the `updated` field from the job state data as it's not accurate and the FE doesn't use it.

Reply:
It might be an idea to remove the `updated` field from the job state data as it's not accurate and the FE doesn't use it.

Reply:
Hmm. Good question. Thinking this through, I think this is a "natural" improvement because if the `state` is super minimal (like doesn't have `"job_input"`, and `_acc_state` __has__ accrued lots of information, like a `"job_input"`, then a superficial merge like `{**self._acc_state, **state}` would trigger a change indication

Reply:
Well ... I can't really think of one either. So I was originally thinking of this (or anything like it)
```
    def __setattr__(self, name, value):
        if name in STATE_ATTRS:
            self._update_state({name: value})
        elif name in JOB_INPUT_ATTRS:
            self._update_state({"job_input": {name: value}})
        elif name in NARR_CELL_INFO_ATTRS:
            self._update_state({"job_input": {"narrative_cell_info": {name: value}}})
        else:
            object.__setattr__(self, name, value)
```
But all the jobs *should* start out with the nested dict `"job_input"` set, and that nested dict *should* never change ...

Reply:
This can be changed back to using `{**self._acc_state, **state}` since we've established there are no cases where the recursive merge would be used.